src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -127,9 +126,8 @@ def _test_flexible_server_mgmt(self, database_engine, resource_group):
             self.cmd('mysql flexible-server create -g {} -l {} --tier MemoryOptimized --sku-name Standard_E2ds_v4 --public-access none'
                      .format(resource_group, location))
 
-        show_output = self.cmd('{} flexible-server show -g {} -n {}'
-                               .format(database_engine, resource_group, server_name), checks=list_checks).get_output_in_json()
-        self.assertIn('subnetArmResourceId', show_output["delegatedSubnetArguments"])
+        self.cmd('{} flexible-server show -g {} -n {}'
```

Comment:
it seems get_output_in_json() is not used

Reply:
Yes, we are not using it anyways.